lib/tool_belt/redmine/project.rb

```diff
@@ -28,7 +28,7 @@ def get_issues(params = {})
     end
 
     def get_issues_for_release(id)
```

Comment:
This now matches the `get_issues_for_version` method. We can probably just remove this and switch all callers. We should also get rid of the release naming in favor of version where it's Redmine terms.

Reply:
Let's do as a follow up issue as I want to make sure use of these configs is accurate.